### PR TITLE
Add colortils.nvim

### PR DESCRIPTION
**Pull request by max397574**
not sure if everything is correct like this
I just copied things from a different pr and modified what I thought had to be modified

### CODE REVIEW

**Comment by max397574** [comment]
@phaazon ?

File: contents/2022/Sep/05.md [hunk 1]
```diff
@@ -24,6 +24,25 @@ People have been busy but… nothing really interesting worth discussing for thi
...
+        <i class="fa-solid fa-book"></i>
+      </span>
+      <span>colortils.nvim</span>
+    </span>
+  </a>
+</h3>
+
+![https://github.com/nvim-colortils/colortils.nvim](https://user-images.githubusercontent.com/81827001/187237256-e8b736cc-17f3-4521-a2ad-3c814b503481.mov)
+
+A neovim plugin for working with colors.
```
Comment (phaazon) [inline contents/2022/Sep/05.md:40]: I think the text should be a bit more description than that, if possible. Thank you in advance.

File: contents/2022/Sep/05.md [hunk 2]
```diff
@@ -24,6 +24,25 @@ People have been busy but… nothing really interesting worth discussing for thi
...
+
+- [GitHub](https://github.com/nvim-colortils/colortils.nvim)
+- [Reddit](https://www.reddit.com/r/neovim/comments/x0s3ku/colortilsnvim_work_with_colors_in_neovim/)
```
Comment (phaazon) [inline contents/2022/Sep/05.md:43]: May you swap those lines? Everywhere else the reddit link comes first.

**Review by phaazon — CHANGES_REQUESTED** [review]

File: contents/2022/Sep/05.md (hunk 1 above)
Comment (max397574) [reply contents/2022/Sep/05.md]: better now @phaazon ?

**Review by phaazon — APPROVED** [review]

File: contents/2022/Sep/05.md [hunk 3]
```diff
@@ -24,6 +24,27 @@ People have been busy but… nothing really interesting worth discussing for thi
 
 ## [New plugins](#new-plugins) {#new-plugins}
 
+h3 id="new-colortils.nvim">
```
Comment (phaazon) [inline contents/2022/Sep/05.md:27]: The tag is messed up but I will fix that.

**Pull request merged**